### PR TITLE
icp-management-ingress digest update 2.5.10

### PR DESCRIPTION
**Pull request by Tirumalavasa**

### CODE REVIEW

File: deploy/olm-catalog/ibm-monitoring-prometheus-operator-ext/1.10.0/ibm-monitoring-prometheus-operator-ext.v1.10.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -485,7 +485,7 @@ spec:
                       - name: PROM_CONF_IMAGE
                         value: quay.io/opencloudio/prometheus-config-reloader@sha256:6d76da1fe18ac9c343dbd104d2763e2918c9ed89b7dac659907286eda9fe0e94
                       - name: ROUTER_IMAGE
-                        value: quay.io/opencloudio/icp-management-ingress@sha256:fedfb66a2c552d6bf1a741dbe42b74aaf0775f8a0618b1f39815474ebc811b7b
+                        value: quay.io/opencloudio/icp-management-ingress@sha256:6641653b351901eb9352e06479d75a4babaf746bcd63381a09a58d9192b0953c
```
Comment (dongwangdw) [inline deploy/olm-catalog/ibm-monitoring-prometheus-operator-ext/1.10.0/ibm-monitoring-prometheus-operator-ext.v1.10.0.clusterserviceversion.yaml:488]: did you update the hash in 1.10.0?
If so, revert it. we do not update release under release protection.

Comment (Tirumalavasa) [reply deploy/olm-catalog/ibm-monitoring-prometheus-operator-ext/1.10.0/ibm-monitoring-prometheus-operator-ext.v1.10.0.clusterserviceversion.yaml]: @dongwangdw , Thanks for pointing it out. I reverted this change.
Can you check the PR now.

**Comment by Tirumalavasa** [comment]
Duplicate of https://github.com/IBM/ibm-monitoring-prometheus-operator-ext/pull/63, closing this PR

**Pull request closed without merging**